src/extension/inpage-script/index.js

```diff
@@ -3,7 +3,11 @@ import { ABORT_PROMPT_ERROR, USER_REJECTED_ERROR } from "~/common/constants";
 import WebLNProvider from "../ln/webln";
 
 if (document) {
-  window.webln = new WebLNProvider();
+  // window.webln is normally loaded onstart (see onstart.js)
```

Comment:
Why do we need that? Is there any reason the `onstart.js` isn't being executed?

Reply:
I don't know. there are certain `if` cases where webln will not be enabled and because it is crucial I thought it does not hurt to do this check again.